github.go

```diff
@@ -76,6 +79,11 @@ func NewGitHubClient(source Source) (*GitHubClient, error) {
 		if err != nil {
 			return nil, err
 		}
+
+	}
+
+	if source.GitHubV4APIURL != "" {
+		clientV4 = githubv4.NewEnterpriseClient(source.GitHubAPIURL, httpClient)
```

Comment:
Looks like this should be using `source.GitHubV4APIURL` - although, see the note above about setting a default